tests/cases/fourslash/quickInfoLink4.ts

```diff
@@ -0,0 +1,9 @@
+/// <reference path="fourslash.ts" />
+
+////const A = 123;
+/////**
+//// * See {@link A | constant A} instead

```

Comment:
looking at the code, it looks like `"| "`, `" |"` and `" | "` will all get skipped. Can you test all 3?